src/ml/neural_net/CMakeLists.txt

```diff
@@ -62,7 +62,13 @@ make_library(tctensorflow SHARED
   tf_model_backend.cpp
 )
 
-set_target_properties(tctensorflow PROPERTIES SUFFIX ".so")
+set(TF_BUILD_DIR ${CMAKE_BINARY_DIR}/src/python/turicreate/toolkits)
```

Comment:
What is this for?

Reply:
The shared library was being made in `turicreate/debug/src/ml/neural_net`. This lines moves it to `turicreate/debug/src/python/turicreate/toolkits`. Then I can simply import it in the python file as `import turicreate.toolkits.libtctensorflow`.